benchmark/torch/maddpg/train.py

```diff
@@ -137,43 +132,33 @@ def train_agent():
                     'model file {} does not exits'.format(model_file))
             agents[i].restore(model_file)
 
-    t_start = time.time()
-    logger.info('Starting...')
+    total_steps = 0
+    total_episodes = 0
+    test_flag = 0
     while total_episodes <= args.max_episodes:
         # run an episode
         ep_reward, ep_agent_rewards, steps = run_episode(env, agents)
         summary.add_scalar('train_reward/episode', ep_reward, total_episodes)
```

Comment:
please add a line to record reward with respect to total_steps
`summary.add_scalar('train/episode_reward_wrt_step', ep_reward, total_steps)`